main.cpp

```diff
@@ -35,89 +35,109 @@ arm.rotateCounterClockwise(); //demonstrates CamelCase
 
 /*
  1)
- Noun:
- action 1:
- action 2:
- action 3:
+ Noun: missile
+ action 1: the missile is fired
+ action 2: the missile locks onto a target
+ action 3: the missile explodes
  */
-
+missile.fire();
+missile.selectTarget();
+missile.explode
 
 /*
  2)
- Noun:
- action 1:
- action 2:
- action 3:
+ Noun: apple
+ action 1: the apple is eaten
+ action 2: the apple is sliced
+ action 3: the apple is thrown
  */
-
+apple.eat();
+apple.slice();
+apple.toss();  //looks like throw is already a defined function
```

Comment:
It's best to not write actions for an object that sound like the object is having something done _**to it**_ 
for example, compare: 
```
Noun: human
action 1: the human eats an apple. 
human.eatApple();
```
That action translates to pseudo-code better and more accurately than "the apple is eaten" `apple.eat();`
`apple.eat();` seems like the apple is consuming something, as the apple is the one "performing" the action.

- [ ] consider changing your noun and actions.